foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle8Platform.java

```diff
@@ -14,12 +14,341 @@
 // Contributors:
 //     06/26/2018 - Will Dazey
 //       - 532160 : Add support for non-extension OracleXPlatform classes
+//     05/06/2019 - Jody Grassel
+//       - 547023 : Add LOB Locator support for core Oracle platform.
+
 package org.eclipse.persistence.platform.database;
 
+import java.lang.reflect.Method;
+import java.security.AccessController;
+import java.security.PrivilegedActionException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Hashtable;
+
+import org.eclipse.persistence.internal.databaseaccess.DatabaseCall;
+import org.eclipse.persistence.internal.databaseaccess.FieldTypeDefinition;
+import org.eclipse.persistence.internal.databaseaccess.Platform;
+import org.eclipse.persistence.internal.databaseaccess.SimpleAppendCallCustomParameter;
+import org.eclipse.persistence.internal.helper.ClassConstants;
+import org.eclipse.persistence.internal.helper.DatabaseField;
+import org.eclipse.persistence.internal.security.PrivilegedAccessHelper;
+import org.eclipse.persistence.internal.security.PrivilegedGetContextClassLoader;
+import org.eclipse.persistence.internal.sessions.AbstractSession;
+import org.eclipse.persistence.logging.SessionLog;
+import org.eclipse.persistence.platform.database.OraclePlatform;
+import org.eclipse.persistence.queries.Call;
+
 /**
  * <p><b>Purpose:</b>
  * Provides Oracle version specific behavior when 
  * org.eclipse.persistence.oracle bundle is not available.
  */
 public class Oracle8Platform extends OraclePlatform {
+    /**
```

Comment:
Add `@Override` annotations to all the methods that allow them.

Reply:
Done